src/transformers/models/character_bert/modeling_character_bert.py

```diff
@@ -0,0 +1,2004 @@
+# coding=utf-8
+# Copyright Hicham EL BOUKKOURI, Olivier FERRET, Thomas LAVERGNE, Hiroshi NOJI,
+# Pierre ZWEIGENBAUM, Junichi TSUJII and The HuggingFace Inc. team.
+# All rights reserved.
```

Comment:
Seeing as some code iof this file s taken from AllenNLP, could you mention them in the copyright of this file?